contents/handbook/people/team-structure/core-experience.md

```diff
@@ -5,13 +5,53 @@ showTitle: true
+is not called "Team App", but "Team Core Experience".
+
+Obviously one team can't do all of this alone. PostHog consists of multiple small teams who are specialised in various
+slices of the app, such as [Team Core Analytics](/handbook/people/team-structure/core-analytics) (analytic queries at scale), 
+[Team Growth](/handbook/people/team-structure/growth-engineering) (feature flags, AB testing), and 
+[Team Platform](/handbook/people/team-structure/platform) (plugins, deployments). 
+
+In addition to working on features ourselves, Team Core Experience provides technical alignment to the rest of the 
+organisation in matters of product design, user experience, and frontend and API tooling.
+
 ## People
 
 [See team structure page](/handbook/people/team-structure/team-structure)
 
 ## Mission
 
-To create the smoothest experience when discovering insights about products and its end users.
+Team Core Experience aims to:
+
+- Implement PostHog's vision of increasing the number of successful products in the world, by building features that will move us closer to said vision.
+- Maintain a coherent strategy for frontend and API development (technologies, tools and methodology).
+- Always be in touch with product leadership and our customers, in order to figure out what works, what doesn't, and what needs to be built next.
+- Actually build those features.
+- Support other teams in PostHog in maintaining a coherently high level of product experience acrosse the board.
+- Push the envelope and give back to the open source community whenever when develop abstractions that could benefit other organisations.
+
+## Goals
+
+With the work above, we aim to solve the following meta-goals:
+
+- Part of our vision: It should be unbelievably easy and intuitive for anyone to use PostHog to get answers to the questions they seek, and even find the answers to questions that had not considered asking.
+
+- To make our tech worthy of the religious following that is emerging around PostHog.
+- Build the next generation of engineers and engineering leaders, who would be running things when we are at 100 developers.
+- Remain a highly flexible team, ready to adapt to changing business requirements.
+- Be part of formalising "the posthog way", and writing a book about it.
+- Our tech stack is rather unique, though pretty powerful. We are also in a really powerful position to drive change in the industry ([data-first frontend](https://kea.js.org/blog/2021/05/14/data-first-frontend-revolution), anyone?). Let's capitalise on that.
+- Zero typescript errors, code covered by logic tests, all scenes in storybook, etc.
+- To do a damn good job at all of the above.
+
+In short, if you want a crash course at how to build large and complicated webapps, at scale, while retaining a maintainable codebase and high customer satisfaction, and touch every part of the stack, this is the team to work with.
```

Comment:
"complicated" sounds a bit as if we were bad at managing these webapps… "complex" maybe?